client/asset/btc/btc.go

```diff
@@ -367,14 +363,20 @@ func BTCCloneWallet(cfg *BTCCloneCFG) (*ExchangeWallet, error) {
 		return nil, fmt.Errorf("error creating BTC RPC client: %v", err)
 	}
 
-	btc := newWallet(cfg, btcCfg, client)
+	feesPerByte := toSatoshi(btcCfg.FallbackFeeRate * 1000)
```

Comment:
bitcoind's `fallbackfee` is BTC/kB, previously with a default of 0.0002, so to get to BTC/byte don't we want to divide by 1000 before converting to sat instead of times 1000?